wheretofindme/models.py

```diff
@@ -51,13 +51,18 @@ class InternetIdentity(models.Model):
     class Meta:
         verbose_name_plural = "Internet Identities"
         ordering = (
+            "seq",
             "created_at",
         )
+        unique_together = [
+            ("user", "seq"),
+        ]
 
     created_at = models.DateTimeField(auto_now_add=True)
     user = models.ForeignKey(User, on_delete=models.PROTECT)
     name = models.CharField(max_length=100, blank=True)
     url = models.CharField(max_length=200, blank=True)
+    seq = models.IntegerField(null=True)
```

Comment:
`seq` isn't an sql reserved word, and I'd prefer not to have `_magic underscores` in field names. I'm okay with `order` otherwise, though, and I know the Django ORM will deal with quoting. Still prefer `order`?

Reply:
Nope, you've convinced me. Stet.